### DISCUSSION TITLE
[@types/chrome] File is not a module

### DISCUSSION THREAD

**Discussion by Prabhudatta-Singh** [DefinitelyTyped/DefinitelyTyped#65141] [Issues with a @types package] [answered]
Hi everyone,
So I am getting this error:

File 'f:/Kritrim Translator/Kritrim-Translator/node_modules/@types/chrome/index.d.ts' is not a module.ts(2306)

when I try to import Chrome: 
`import * as chrome from 'chrome';`

I have installed it twice, normal install 
`npm i @types/chrome` and as a dev dependency `npm install --save-dev @types/chrome` but I still get this error.

It was suggested somewhere to add this to the tsconfig.json file:
`"types": [
  "chrome"
]` but still no fix

I think this is a related issue but I am not sure, all my parameters get the error:
Parameter 'notificationId' implicitly has an 'any' type.ts(7006),
Parameter 'newState' implicitly has an 'any' type.ts(7006)

Is this normal or happening because of the first issue?

Am I making any mistakes here? Or should I add any other piece of code to make it work? 
I am using Sveltekit btw.

[a comment by an automated account is not shown]

**Comment by fregante** [comment, marked as answer]
My guess is that your tsconfig is using the "wrong" `moduleResolution` setting. This is not related to `@types/chrome` so you might want to ask in a different forum or try `moduleResolution: "Node"`

**Reply by Prabhudatta-Singh** [reply]
Thanks for the help.
I will ask somewhere else.
